libcylinder/src/key.rs

```diff
@@ -93,6 +101,14 @@ impl PrivateKey {
     }
 }
 
+impl std::fmt::Debug for PrivateKey {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        f.debug_struct("PrivateKey")
+            .field("bytes", &self.as_hex())
```

Comment:
Is logging the complete key a security concern? Especially if this is just for testing purposes

Reply:
I guess the debug/display implementations aren't needed for the tests; I can remove them for private key if this is a security issue.

Reply:
Removed the display/debug impls for private key and updated the commit